main/src/ca/uwaterloo/flix/language/phase/Redundancy.scala

```diff
@@ -1250,6 +1303,7 @@ object Redundancy {
                                    sigSyms: ConcurrentHashMap[Symbol.SigSym, Unit],
                                    effSyms: ConcurrentHashMap[Symbol.EffectSym, Unit],
                                    enumSyms: ConcurrentHashMap[Symbol.EnumSym, Unit],
+                                   structSyms: ConcurrentHashMap[Symbol.StructSym, Unit],
                                    caseSyms: ConcurrentHashMap[Symbol.CaseSym, Unit])
```

Comment:
caseSyms are related to enumSyms, so if we could make the order: enumSyms, caseSyms, structSyms.